runtime/vm/JFRConstantPoolTypes.cpp

```diff
@@ -1322,6 +1404,36 @@ VM_JFRConstantPoolTypes::addModuleRequireEntry(J9JFRModuleRequire *moduleRequire
 
 }
 
+void
+VM_JFRConstantPoolTypes::addModuleExportEntry(J9JFRModuleExport *moduleExportData)
+{
+	U_32 exportedPackageIndex = addPackageEntry(moduleExportData->fromModule, moduleExportData->exportedPackage);
+	// Skip this entry if no class from the package has been loaded
+	if (0 == exportedPackageIndex) {
+		return;
+	}
+
+	ModuleExportEntry *entry = (ModuleExportEntry *)pool_newElement(_moduleExportTable);
```

Comment:
entry does not appear to be used anywhere.

Reply:
It adds a new entry to the pool which will be iterated later in https://github.com/eclipse-openj9/openj9/pull/22140/files#diff-853d916baa643ca7da0874e20cebf3b757d1cda21396ce848610745686cc1e36R414.

Reply:
I see. Can you please add a comment to explain that?